clients/fides-js/src/fides-tcf.ts

```diff
@@ -225,14 +223,9 @@ const init = async (config: FidesConfig) => {
   });
   Object.assign(_Fides, updatedFides);
 
-  // Dispatch the "FidesInitialized" event to update listeners with the initial
-  // state. Skip if we already initialized due to an existing cookie.
-  // For convenience, also dispatch the "FidesUpdated" event; this allows
-  // listeners to ignore the initialization event if they prefer
-  if (isNewFidesCookie(cookie)) {
-    dispatchFidesEvent("FidesInitialized", cookie, config.options.debug);
-  }
-  dispatchFidesEvent("FidesUpdated", cookie, config.options.debug);
+  // Dispatch the "FidesInitialized" event to update listeners with the initial state.
+  // TODO PROD-1280: Suppress duplicate event if the cookie state is unchanged from an early initialization?
```

Comment:
I'm really not sure about this yet. I like the _simplicity_ of this code right now, but the double-triggering of the event doesn't "feel" great. Looking for opinions 😄